src/psyclone/psyir/transformations/psy_data_trans.py

```diff
@@ -258,7 +258,11 @@ def apply(self, nodes, options=None):
         position = node_list[0].position
         root = node_list[0].root
 
-        # create a memento of the root and the proposed
+        # We always use the outermost symbol table so that any name clashes
```

Comment:
This looks like a change that should already be on master. Please could you check.

Reply:
Resolved an issue with the comment but everything else should be fine now.